math/dec.go

```diff
@@ -576,3 +576,60 @@ func MaxDec(x, y Dec) Dec {
 	}
 	return y
 }
+
+// MustAdd performs addition between two Dec values.
+// If an error occurs during the addition, it panics.
+func (x Dec) MustAdd(y Dec) Dec {
+	z, err := x.Add(y)
+	if err != nil {
+		panic("MustAdd failed: " + err.Error())
+	}
+	return z
+}
+
+// MustSub performs subtraction between two Dec values.
+// If an error occurs during the subtraction, it panics.
+func (x Dec) MustSub(y Dec) Dec {
+	z, err := x.Sub(y)
+	if err != nil {
+		panic("MustSub failed: " + err.Error())
+	}
+	return z
+}
+
+// MustMul performs multiplication between two Dec values.
+// If an error occurs during the multiplication, it panics.
+func (x Dec) MustMul(y Dec) Dec {
+	z, err := x.Mul(y)
+	if err != nil {
+		panic("MustMul failed: " + err.Error())
+	}
+	return z
+}
+
+// MulInt multiplies a Dec value by an int64.
+// If an error occurs during the multiplication, it panics.
+func (x Dec) MulInt(y int64) Dec {
```

Comment:
i think we should add `Must` if we're going to panic here. otherwise, it should return the error.